### PR TITLE
Feature/add stuff for menu

### PR DESCRIPTION
**Pull request by robineng**

### CODE REVIEW

**Comment by mjmonline** [comment]
Awesome!

File: src/Atoms/Icon/__snapshots__/Icon.stories.storyshot [hunk 1]
```diff
@@ -2668,6 +2705,41 @@ exports[`Storyshots Atoms | Icon Available Icons 1`] = `
...
+          }
+        }
+      >
+        <svg
+          aria-hidden="true"
+          className="c3"
+          focusable="false"
+          preserveAspectRatio="xMidYMid meet"
+          role="presentation"
+          size={5}
```
Comment (mjmonline) [inline src/Atoms/Icon/__snapshots__/Icon.stories.storyshot]: I know your PR didn't cause this but could you please omit size prop so it doesn't leak to the html? 🙏

Comment (mjmonline) [reply src/Atoms/Icon/__snapshots__/Icon.stories.storyshot]: if you're unsure how, please just create a issue then. Thanks in advance

Comment (robineng) [reply src/Atoms/Icon/__snapshots__/Icon.stories.storyshot]: Fixed!

**Pull request merged**